carla_integration/drivers.launch

```diff
@@ -37,4 +37,10 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="mock_gnss"           value="$(eval 'gnss' in arg('mock_drivers').split())"/>
   <arg name="mock_lidar"          value="$(eval 'lidar' in arg('mock_drivers').split())"/>
```

Comment:
I think you can remove all the mock driver stuff. You would never use a mock driver in a simulation context.

Reply:
They are needed for health_monitor. I think gps, controller and lidar are the minimum requirements for the health_monitor node.